website/careers/index.html

```diff
@@ -0,0 +1,320 @@
+
+{% block header_content %}
+  <section class="text-left text-white flex flex-col items-start xl:w-full max-w-6xl mx-auto pl-8 pr-8">
+    <h1 class="font-4xl md:font-hero font-thin md:mt-10 mb-8">{{ self.page_title() }}</h1>
+    <p class="font-lg md:font-xl tracking-wide leading-relaxed max-w-6xl mb-10 mt-0">
+      {{ self.page_desc() }}
+    </p>
+  </section>
+{% endblock %}
+
+{% block content %}
+  <section class="pt-20 pb-0 flex justify-between items-center pr-8 pl-8 overflow-hidden">
+    <section class="flex flex-col justify-between mx-auto w-full max-w-6xl">
+        <h3 class="header-section">
+          <span>{{ svg('envelope') }}</span>
+          <aside class="flex flex-col">
+            {{ _('What is Thunderbird?')}}
+            <small class="font-normal font-md tracking-normal normal-case">
+              {{ _('Software made to make email easier.') }}
+            </small>
+          </aside>
+        </h3>
+        <p class="font-lg tracking-wider leading-loose mb-6">
+          {{ _('Thunderbird is a free and open source email, newsfeed, chat, and calendaring client, that’s easy to set up and customize. One of the core principles of Thunderbird is the use and promotion of open standards - this focus is a rejection of our world of closed platforms and services that can’t communicate with each other. We want our users to have freedom and choice in how they communicate.') }}
+        </p>
+    </section>
+  </section>
+
+  <section id="getstarted" class="mt-20 mb-14 flex max-w-6xl mx-auto justify-center items-center pl-8 pr-8">
+    <span class="header-line flex-1"></span>
+    <h2 class="header-section pl-4 pr-4 mb-0 text-center">{{ _('Open Positions')}}</h2>
+    <span class="header-line flex-1"></span>
+  </section>
+
+  <section class="pt-10 pb-10 flex flex-col max-w-6xl mx-auto justify-between pl-8 pr-8">
+    <aside>
+      <h3 class="header-section">
+        <span>{{ svg('developer') }}</span>
+        <aside class="flex flex-col">
+          {{ _('Thunderbird Release Engineer / Web Infra Engineer')}}
+          <small class="font-normal font-md tracking-normal normal-case">
+            {{ _('Full Time / Remote') }}
+          </small>
+        </aside>
+      </h3>
+      <p class="font-lg tracking-wider leading-loose mb-5 p-links-blue">
+        {{ _('Thunderbird is looking for a multi-skilled self-starter to work partly on release engineering, and partly on operating the cloud based web infrastructure around Thunderbird. You will be working in close cooperation with our release engineer, infra engineer and other staff, as well as with community members.') }}
+      </p>
+      <p class="font-lg tracking-wider leading-loose mb-6 p-links-blue">
+        {{ _('This is a <strong>remote</strong>, <strong>full time</strong> position. You are expected to have excellent written communication skills and coordinate your work over email, video conferencing, IRC, and Bugzilla.<br/>
```

Comment:
Should this be in its own <p> tag? The <br> does not actually produce a separate paragraph, it looks like this:

![image](https://user-images.githubusercontent.com/479671/73288605-28a25c00-41b0-11ea-9ce0-b7469f099be7.png)

Reply:
That's how it was in the DOC Ryan sent me. I simply respected how it was written. I don't have any preference for this matter

Reply:
Let's make it look nice, the formatting in the doc is likely just not right.